Validators Testnet Docs (#190)
helium/docs: docs/mine-hnt/validators/requirements.mdx

---
id: requirements
title: Validator Requirements
hide_title: true
sidebar_label: Technical Requirements
slug: /mine-hnt/validators/requirements
---

import useBaseUrl from "@docusaurus/useBaseUrl";

# Technical Requirements

Based on early tests, validator node requirements include:
* Operating system: Ubuntu 20.04 LTS (if running from source)
* CPU: Intel processor 3.0 GHz or above with 2 cores or equivalent
* Memory: 8 GB or above
* Storage: 256 GB or more 

> * If using a cloud provider (recommended) an AWS EC2 instance T2 large, xlarge or equivalent

Reviewer: ```suggestion
* If using AWS, a T2 large or xlarge EC2 instance or equivalent is recommended. If using another cloud provider, like GCP, Azure, or Digital Ocean, an instance with the same capabilities should be the target.
```

> * Stable IP and few ports (currently 2154 and eventually port 443) open to internet

Reviewer: ```suggestion
* Stable IP with ports `2154` and `443` open to internet
```

> * Running on stable network connections (without things like proxies, NAT, etc.), load is largely symmetrical when producing blocks, so good upstream recommended.

Reviewer: ```suggestion
* Running on stable network connections free of things like proxies, NAT, etc. The load is largely symmetrical when producing blocks, so good upstream connectivity recommended.
```
